crates/router/src/connector/bankofamerica/transformers.rs

```diff
@@ -330,21 +330,32 @@ impl
     From<(
         &BankOfAmericaRouterData<&types::PaymentsAuthorizeRouterData>,
         Option<PaymentSolution>,
+        Option<String>,
     )> for ProcessingInformation
 {
     fn from(
-        (item, solution): (
+        (item, solution, network): (
             &BankOfAmericaRouterData<&types::PaymentsAuthorizeRouterData>,
             Option<PaymentSolution>,
+            Option<String>,
         ),
     ) -> Self {
+        let commerce_indicator = match network {
+            Some(card_network) => match card_network.as_str() {
```

Comment:
Convert these to lower case and then match. And need not construct String in every case, can do .to_string() while returning

Reply:
Done